client/my-sites/hire-a-pro/view.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import MarketingPageFeature from 'components/marketing-page-feature';
+import MarketingPageHeader from 'components/marketing-page-header';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { recordTracksEvent as recordTracksEventAction } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	recordTracksEvent: typeof recordTracksEventAction;
+}
```

Comment:
for the purpose of showing where these are expected to come from it may be helpful to indicate that they are `ConnectedProps`

this prop will be removed from the external interface by `connect()` so when we are using this in some parent component it shouldn't be on the auto-complete list.

so far the `Props` interfaces have all basically been "these are the props that you will see in the auto-complete when using this component" and I think that has value

thoughts?